### PR TITLE
Update house database models

### PR DESCRIPTION
**Pull request by tiusty**
Updating the house database model to be cleaner.

### CODE REVIEW

File: houseDatabase/models.py [hunk 1]
```diff
@@ -25,135 +101,102 @@ class BuildingExteriorAmenities(models.Model):
...
-    def __str__(self):
-        return self.address
-
-    def get_remarks(self):
-        return self.remarks
-
-    def get_listing_no(self):
-        return self.listing_no
+    _remarks = models.TextField(default="")
+    _listing_number = models.IntegerField(default=-1)
```
Comment (tiusty) [inline houseDatabase/models.py]: @seanrayment This was listing_no before, so I assumed it was supposed to be listing_number? Let me know if that is wrong or right

File: houseDatabase/models.py [hunk 2]
```diff
@@ -25,135 +101,102 @@ class BuildingExteriorAmenities(models.Model):
...
-    def get_washer_dryer_in_building(self):
-        return self.washer_dryer_in_building
 
-    def get_elevator(self):
-        return self.elevator
+class RentDatabase(MLSpinData, BuildingExteriorAmenities, InteriorAmenities, HomeBase):
+    """
+    This model stores all the information associated with a home
+    """
+    _apartment_number = models.CharField(max_length=200)
```
Comment (tiusty) [inline houseDatabase/models.py:188]: @seanrayment This was apartment_no before, I assumed it was supposed to be apartment_number, let me know if that is right or wrong

Comment (seanrayment) [reply houseDatabase/models.py:188]: That is correct.

**Review by tdefroscia — APPROVED** [review]
All these changes look good to me.

**Review by seanrayment — APPROVED** [review]

**Review by tdefroscia — APPROVED** [review]

**Pull request merged**